src/Symfony/Component/Console/Style/OutputStyle.php

```diff
@@ -141,13 +152,4 @@ public function isDebug()
     {
         return $this->output->isDebug();
     }
-
-    protected function getErrorOutput()
```

Comment:
removing a protected method is a BC break.

Reply:
Thank you, reverted, can't remember why I've removed this.